sportsdataverse/wbb/wbb_fox_ext.py

```
@@ -0,0 +1,535 @@
+    Example:
+        Fetch a game's plays::
+
+            from sportsdataverse.wbb import fox_wbb_pbp
+            df = fox_wbb_pbp("...")
+    """
+    raw = fox_get(f"{_SPORT}/event/{game_id}/data", **kwargs)
+    return frame(parse_period_pbp(raw, game_id), return_as_pandas) if return_parsed else raw
+
+
+@overload
+def fox_wbb_boxscore(
```

Flake8 (7.3.0)

[error] multiple statements on one line (def)

(E704)